packages/squiggle-lang/src/rescript/Reducer/Reducer_Expression/Reducer_Expression_T.res

```diff
@@ -1,5 +1,15 @@
 open ReducerInterface.ExpressionValue
 
+/*
+  An expression is a Lisp AST. An expression is either a primitive value or a list of expressions.
```

Comment:
reducer is Lisp-like. However expressions are really Lisp-AST. I am not joking about that. There are more than one way of implementing it and this is one of them. I keep referring to it because forgetting that they are Lisp-AST even I can get in the wrong direction. If one they I forget that and manipulate it just as an expression tree then I am in the wrong direction.